docs/user/using_the_client.md

```diff
@@ -155,10 +171,11 @@ rucio -a=\<rucio_account_name\> \
   -v \
   whoami
 ```
+If you are specifying the scope all the required scope is needed . (at minimum `openid profile` but can be more if configured on server. Please check with rucio rucio provider)
```

Comment:
This is a bit confusing: the first half says that if the scope is specified then that's all that's needed, but then we say that we require the openID profile at minimum and then possibly some other things

Reply:
Yes I agree, This is confusing. Let me change a thing in main PR and try to simplify this.

Reply:
Added a new commit in main PR and revised this.